pattern_library/views.py

```diff
@@ -58,6 +59,7 @@ class RenderPatternView(TemplateView):
     http_method_names = ('get',)
     template_name = get_pattern_base_template_name()
 
+    @xframe_options_sameorigin
```

Comment:
Then this line becomes:
`@method_decorator(xframe_options_sameorigin)`

Reply:
Thanks @ababic! Have made the changes you suggest.